packages/docs/package.json

```diff
@@ -6,7 +6,8 @@
     "docusaurus": "docusaurus",
     "start": "docusaurus start --no-open",
     "dev": "yarn start",
-    "build": "docusaurus build",
+    "build:public": "rm -rf public && mkdir public && cp -r build public/",
```

Comment:
why is this needed on top of the docusaurus build?

Reply:
Sorry this was a test, I will remove it